frontend/src/components/volunteer/volunteer.css

```diff
@@ -0,0 +1,95 @@
+* {
```

Comment:
This CSS should be moved into a `components/static/index.css` file to avoid duplication.